.storybook/main.ts

```diff
@@ -19,13 +19,33 @@ export default {
   docs: {
     autodocs: true,
   },
+  core: {
+    builder: '@storybook/builder-webpack5',
+  },
   webpackFinal: async config => {
-    config?.module?.rules?.push({
+    const imageRule = config.module?.rules?.find(rule => {
+      const test = (rule as { test: RegExp }).test
```

Comment:
if you can add a small comment about the reason why excluding the svg from imageLoader, not blocking, nice to have

Reply:
@alexandre-combemorel I will add it here: https://github.com/scaleway/ultraviolet/pull/3309

To answer here: we want to remove any default configuration that storybook has with webpack and apply our own rule with svgr